web-interface/resource-modules/networking-wizard-fields-web.adoc

```diff
@@ -15,6 +15,9 @@
 |NETWORK CONFIGURATION
 |List of available *NetworkAttachmentDefinition* objects.
 
+|BINDING METHOD
+|The definition of the link:../creating-virtual-machines/interfaces-and-networks.html#frontend[binding method]. For pod networking, the possible values are `masquerade`, `bridge` and `sriov`. For the other networks the possible values are `bridge` and `sriov`.
```

Comment:
To remain consistent with the wording of the other drop-down descriptions, I'd go with "List of available binding methods."

nitpick: Add a comma after "bridge".

Reply:
done

Reply:
Would it be appropriate to include links to documents that explain these binding methods?